chapter05/red/exercise3.pl

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/perl
+
+use warnings;
+use strict;
+
+my $fn = shift @ARGV // 'coconet.dat';
+open my $fh, '<', $fn;
```

Comment:
And it doesn't need to be, the `coconet.dat` format only includes ASCII characters.